### DISCUSSION TITLE
Game won't register any Quest2 Inputs , nor mouse/keyboard + Xbox 360 Controller

### DISCUSSION THREAD

**Discussion by DardoJS89** [Okabintaro/SubmersedVR#74] [Q&A]
Hey bossman, 
Thank you for your contributions to the game. Your mod looks like what the original developers should have put together themselves. 
I have not been able to get the game to let me get passed the Main Menu when loading it with your mod. 
My Oculus Quest controllers do not register any inputs, and the mouse/Keyboard along with my Xbox Controller don't work either however they at least do trigger an error log on BepinEx that reads the following for both each time:

 
 [Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
FPSInputModule.OnApplicationFocus (System.Boolean focusStatus) (at <e1b9598e579944b4822a0de2b2ebc567>:0)

It might be worth noting that the version I have is being run via the Microsoft Store, with SteamVR opened up. The game does open up in VR, but as I stated above , errors are displayed upon button press. I've faced issues in the past with VREnhancements, such as the FPSInputModule error being triggered when I selected "Gaze Based Cursor" in the Options menu. When I boot up the game in non VR I can see that the mod shows up in the Subnautica Mod Manager. Hope this helps somehow.

Keep up the good work, your mod looks awesome (for those who can use it).

Best regards

**Comment by Okabintaro** [comment]
Unfortunately the Microsoft Version is outdated and can't work with the mod which requires the Living Large Update. ([You are not the first with that issue](https://github.com/Okabintaro/SubmersedVR/issues/73#issuecomment-1685369052)) Unfortunately I can't do much about that.

**Reply by DardoJS89** [reply]
Hey Okabintaro!
The Living Large update was actually first released onto the Microsoft Store version of the game, before all others. I guess they must have some other discrepancy thats keeping SubmersedVR from working on the platform. I know with WindowsApps alot of gamefiles tend to be locked behind the ownership wall, which I think could be the real issue. This being one of the main reasons why Vortex won't work with MS Store version.
Best of luck with the rest of the updates, I'll be keeping an eye out for Submersed for the months and years to come.

Thanks again.

**Reply by Okabintaro** [reply]
Ah yeah they released it half baked. Like they added the features but didn't add the performance and VR Improvements. We had a discussion about it on discord a couple of months ago: 
https://discord.com/channels/747967102895390741/754030896130031736/1112466712944726227

**Reply by DardoJS89** [reply]
Hey Okabintaro!
I switched on over to Steam and was able to use your mod. I love how immersed I am now, this is so great! I was wondering if I would be able to use the Mouse/Keyboard or an Xbox Controller at all whilst playing with the Oculus Controllers. The reason I ask is because I am having a hard time finding out how I can pause the game so I can save the game. 
This feels so freaking legit man, thank you for bringing this to the Subnautica world, you are a LEGEND.

Best,

**Reply by DardoJS89** [reply]
Found the pause menu in the PDA. That answers that, but I'm still curious about the Xbox Controller / Keyboard support, as I'm not sure how to switch between the Cameras in the Scanner room + certain buttons on the keyboard that have been designated for things such as (Nightvision Mode = P ) and the like don't exactly have usability given the limited button prompts..... otherwise, WOW, this has just become my #1 favorite VR game thanks to YOU. Considering the state of VR prior to you and IWhoI, its insane. Bravo, I salute you.
